crypto/engine/eng_devcrypto.c

```diff
@@ -624,8 +625,10 @@ void engine_load_devcrypto_int()
     prepare_digest_methods();
 #endif
 
-    if ((e = ENGINE_new()) == NULL)
+    if ((e = ENGINE_new()) == NULL) {
```

Comment:
Okay, as is, but I would have done this:

```
if ((e = ENGINE_new()) == NULL
          || !ENGINE_set_destroy_function(e, devcrypto_unload))
      {
           close(cfd);
           return;
      }
```

Reply:
Ah!  Sure, I can do that.  Nice :-)